examples/FasterRCNN/train.py

```diff
@@ -119,7 +119,7 @@
         starting_epoch=cfg.TRAIN.STARTING_EPOCH
     )
     if is_horovod:
-        trainer = HorovodTrainer(average=False)
+        trainer = HorovodTrainer(average=False, aggregation_frequency=2)
```

Comment:
Not a big need, but maybe make this be read from the config file.

Reply:
gonna hold off for now, should be easy enough to do later if we want